subsys/nrf_security/Kconfig.psa

```diff
@@ -88,6 +88,10 @@ config PSA_ITS_ENCRYPTED
 	help
 	  Enables authenticated encryption for PSA Internal Trusted Storage files
 
+config PSA_CRYPTO_DRIVER_ALG_PRNG_TEST
+	bool
+	help
+		Internal option used for testing.
```

Comment:
Nit: You can expand just a bit here if you want.

Reply:
More nit: Indent: 1 tab + 2 spaces